components/admin_real/desktop/Body.jsx

```diff
@@ -20,7 +20,7 @@ export default function Body({ view }) {
     default:
       return (
         <BodyFrame>
-          <ReservationTable />
+          <ReservationTableContextProvider />
```

Comment:
This is not usually how you use a context provider. 

Look at how its done in the `_app.js` file.

You would do this instead here:
```
<ContextProvider>
    <ReservationTable>
</ContextProvider>
```